Kudu.Core/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="LibGit2Sharp" version="0.20.0.0" targetFramework="net45" />
```

Comment:
I'd strongly suggest you to update to `0.20.1` with regards to http://git-blame.blogspot.fr/2014/12/on-cve-2014-9390-and-git-221.html